R/from.R

```diff
@@ -59,7 +67,8 @@
 #' import::from(parallel, makeCluster, parLapply)
 #' import::into("imports:parallel", makeCluster, parLapply, .from = parallel)
 from <- function(.from, ..., .into = "imports", .library = .libPaths()[1L],
-                 .character_only = FALSE)
+                 .all=FALSE, .except=character(),
```

Comment:
l would suggest to change this to

```suggestion
                 .all=(length(.except) > 0), .except=character(),
```

so using `.except` implied `.all=TRUE` by default.
I don't see any use case for `.except` with `.all=FALSE` (and even if there is one, this change would still allow it by specifiying both parameters explicitely), so this will safe some code in most cases.

Also, personally I find

```R
import::from(foo, .except=c("bar", "bat"))
```

equally understandable as

```R
import::from(foo, .all=TRUE, .except=c("bar", "bat"))
```

but faster to parse.

Reply:
That’s a very nice touch! I hadn’t thought of that, I’ll update the pull request to include it.